generators/app/index.js

```diff
@@ -70,24 +70,32 @@ class GeneratorReact extends Generator {
     this.log('Copying base project files...');
     return Promise.resolve()
       .then(createPackageJson.bind(this))
-      .then(createJSConfig.bind(this))
-      .then(copyFiles.bind(this))
+      .then(copyAllFiles.bind(this))
       .then(this.steps.writing.bind(this));
   }
 
   install() {
     this.log('Installing...');
     return new Promise(resolve => {
-      this.spawnCommand('npm', ['install']).on('close', () => resolve());
+      this.spawnCommand('npm', ['install'], { cwd: `${process.cwd()}/${this.projectName}` }).on('close', () =>
+        resolve()
+      );
     });
   }
 
   async end() {
     this.log('Ending...');
+    await linterAutofix
```

Comment:
Why are we ignoring?

Reply:
It was reporting the error two times, also finishing the bootstrap on that step.
This way you'll have an error but the rest will continue.